usr/src/uts/common/os/logsubr.c

```diff
@@ -263,7 +263,7 @@ log_init(void)
 	printf("\rSunOS Release %s Version %s %u-bit\n",
 	    utsname.release, utsname.version, NBBY * (uint_t)sizeof (void *));
 	/* NOTE: We might switch to BOOTBANNER after this. */
-	printf("Copyright 2022-2024 MNX Cloud, Inc.\n");
```

Comment:
Probably a good idea, but I'm pretty sure the image I just built locally will have 2022-2025 already in it thanks to bootbanner.